packages/kolibri-common/components/SearchFiltersPanel/index.vue

```diff
@@ -21,94 +21,93 @@
       v-if="windowIsLarge || !currentCategory"
       :class="windowIsLarge ? '' : 'drawer-panel'"
     >
-      <div v-if="Object.keys(availableLibraryCategories).length">
-        <div v-if="!accordion">
-          <!-- search by keyword -->
-          <h2 class="title">
-            {{ $tr('keywords') }}
-          </h2>
-          <SearchBox
-            key="channel-search"
-            ref="searchBox"
-            :placeholder="coreString('findSomethingToLearn')"
-            :value="value.keywords || ''"
-            @change="val => $emit('input', { ...value, keywords: val })"
+      <div v-if="!accordion">
+        <!-- search by keyword -->
+        <h2 class="title">
+          {{ $tr('keywords') }}
+        </h2>
+        <SearchBox
+          key="channel-search"
+          ref="searchBox"
+          :placeholder="coreString('findSomethingToLearn')"
+          :value="value.keywords || ''"
+          @change="val => $emit('input', { ...value, keywords: val })"
+        />
+        <h2 class="section title">
```

Comment:
This `<h2>` and the next `<div>` should be wrapped in another div that is conditionalized using `v-if="Object.keys(availableLibraryCategories).length"`.

Reply:
got it

Reply:
I think this will break it

Reply:
Break what? The issue was that the condition was being applied to the entire filters panel previously, so when you were importing resources that had no categories set on them, that was causing the reported issue (that they happened to be exercises was a red herring, I think). Restoring the condition to its correct place _just_ wrapping the categories title and div will restore the previous behaviour.

Reply:
No- I was wrapping it wrongly . It's all good now.